test/test_samples.py

```diff
@@ -18,10 +18,17 @@ def test_3d_samples_response():
     with open('3d_fragility.json', 'r') as f:
     # with open('Galveston_wood_poles.json', 'r') as f:
         frag = json.load(f)
-        params = {'sample_interval':0.5}
+        params = {'sample_size': 2, 'refresh': True}
         response = requests.post('http://127.0.0.1:5000/plotting/api/samples', json=frag, params=params)
         print(response.content)
 
 
+def test_multihazard_samples_response():
+    with open('multihazard_fragility.json', 'r') as f:
+        frag = json.load(f)
+        params = {'sample_size': 2, 'refresh': True}
+        response = requests.post('http://127.0.0.1:5000/plotting/api/samples', json=frag, params=params)
```

Comment:
when I tried to run, this line made an error of
requests.exceptions.ConnectionError: HTTPConnectionPool(host='127.0.0.1', port=5000): Max retries exceeded with url: /plotting/api/samples?sample_size=2&refresh=True (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000023A37335A80>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
python-BaseException

Reply:
yes you need to have a service up running first. You can follow the README to get it running